brianlong/www.validators.app
# 178998318_add_average_to_batches

#### What's this PR do?
- add average to batches
- backfill averages for batch

#### How should this be manually tested?
- Part 1/2
- Run migration
- Run script `rails r script/fill_average_skipped_slot_percent_in_batches.rb`
- Check if batch entities got new values

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/178998318

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
